js/gradient-generator.js

```diff
@@ -71,17 +69,22 @@ const transitionGradient = (onLoad) => {
 	applyChange();
 }
 const applyChange = () => {
-	let cssGradient = `linear-gradient(45deg,${stopOne},${stopTwo})`;
-	let outer = document.querySelector('.outer');
-	let currentGradientState = {
+	const cssGradient = `linear-gradient(45deg,${stopOne},${stopTwo})`;
+	const outerElem = document.querySelector('.outer');
+	const currentGradientState = {
 		gradIndex: firstGrad ? this.gradIndex = 'one' : this.gradIndex = 'two',
-		gradValue: firstGrad ? this.gardValue = stopOne : this.gardValue = stopTwo
+		gradValue: firstGrad ? this.gardValue = stopOne : this.gardValue = stopTwo // gardValue? or gradValue ???
 	}
+	// is this what you want to do?
+	// const currentGradientState = {
+	// 	gradIndex: firstGrad ? 'one' : 'two',
+	// 	gradValue: firstGrad ? stopOne : stopTwo
+	// }
 	if (supportsCssVars) {
 		document.documentElement.style.setProperty(`--gradient-${currentGradientState.gradIndex}`,`${currentGradientState.gradValue}`);
 	} else {
-		outer.style.backgroundImage = cssGradient;
+		outerElem.style.backgroundImage = cssGradient;
 	}
-	outer.classList.add('active');
-	firstGrad = !firstGrad;
+	outerElem.classList.add('active');
+	firstGrad = !firstGrad; // this is weird, what's it
```

Comment:
To spread/update css gradient values evenly I toggle first vs. second gradient each time value is updated. This way once gradient in left , bottom corner and then value of the gradient in the top right.